Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 edition = "2021"
 name = "holochain_scaffolding_cli"
-version = "0.1.11"
```

Comment:
We shouldn't make version changes on this branch please, the versions need to be maintained separately on the release branches and changing it here risks mistakes in the back-port

Reply:
Ok will revert